requirements-dev.txt

```diff
@@ -21,14 +21,15 @@ flake8==3.7.9             # via -r requirements-dev.in
 freezegun==0.3.15         # via -r requirements-dev.in
 hypothesis==5.5.4         # via -r requirements-dev.in
 idna==2.9                 # via -c requirements.txt, requests
+importlib-metadata==1.6.0  # via jsonschema, pluggy, pytest
```

Comment:
This got removed in a recent PR - wonder why it's back?

Reply:
🤔

Reply:
> Skip dependency importlib_metadata on PY 3.8

https://github.com/Julian/jsonschema/commit/0e6c684a51c5593087b8bb419d54f79db591e7d6

Reply:
Not sure what to do about that

Reply:
Let's just remember to keep an eye on it for now? I want to get this PR merged.

I will do some research also.